painttiming.bs

```diff
@@ -27,6 +27,30 @@ urlPrefix: https://www.w3.org/TR/hr-time-2/#idl-def-domhighrestimestamp; spec: H
     type: typedef; text: DOMHighResTimeStamp
 urlPrefix: https://www.w3.org/TR/CSS2/visuren.html; spec: CSS-2;
     type: dfn; url: #viewport; text: viewport
+urlPrefix: https://www.w3.org/TR/CSS22/visufx.html; spec: CSS-2;
+    type: dfn; url: #propdef-visibility; text: visibility;
+urlPrefix: https://www.w3.org/TR/css-color-3; spec: CSS-COLOR-3;
+    type: dfn; url: #opacity; text: opacity;
+urlPrefix: https://www.w3.org/TR/cssom-view/; spec: CSSOM-VIEW-1;
+    type: dfn; text: getBoundingClientRect; url: #dom-element-getboundingclientrect
+urlPrefix: https://dom.spec.whatwg.org/; spec: DOM
+    type: dfn; text: text node; url: #text
+urlPrefix: https://www.w3.org/TR/css-images-3/#url-notation; spec: CSS-IMAGES-3;
+    type: dfn; text: loaded; url: #typedef-image
+urlPrefix: https://www.w3.org/TR/SVG2/render.html; spec: CR-SVG2
+    type: dfn; url: #Rendered-vs-NonRendered; text: svg element with rendered descendants;
+urlPrefix: https://www.w3.org/TR/css-text-3; spec: CSS-TEXT-3;
+    type: dfn; text: white-space; url: #white-space;
+urlPrefix: https://www.w3.org/TR/css-backgrounds-3/; spec: CSS-BACKGROUNDS-3;
+    type: dfn; text: background-image; url: #propdef-background-image;
+urlPrefix: https://www.w3.org/TR/css-backgrounds-3/; spec: CSS-MASKING-1;
+    type: dfn; text: mask-image; url: #the-mask-image;
+urlPrefix: https://html.spec.whatwg.org/
```

Comment:
Can we use the multipage version? This version is incredibly slow because it loads the whole spec at once.

Reply:
sure!